pandas/core/generic.py

```diff
@@ -4934,19 +4938,21 @@ def bfill(self, axis=None, inplace=False, limit=None, downcast=None):
             other views on this object (e.g. a column from a DataFrame).
             Returns the caller if this is True.
         limit : int, default None
-            Maximum size gap to forward or backward fill
+            Maximum size gap to forward or backward fill.
         regex : bool or same types as ``to_replace``, default False
             Whether to interpret ``to_replace`` and/or ``value`` as regular
             expressions. If this is ``True`` then ``to_replace`` *must* be a
             string. Alternatively, this could be a regular expression or a
             list, dict, or array of regular expressions in which case
             ``to_replace`` must be ``None``.
-        method : string, optional, {'pad', 'ffill', 'bfill'}
+        method : string, optional, {'pad', 'ffill', 'bfill'}, default is 'pad'
             The method to use when for replacement, when ``to_replace`` is a
             scalar, list or tuple and ``value`` is None.
+        axis : None
+            Deprecated.
```

Comment:
Warning says this will be removed in v0.13? Woof...I guess OK to document for this change but should have a follow up change to actually go ahead and remove - care to take a stab at that?

Reply:
@WillAyd where is this warning?

Reply:
Just a couple of lines into the function definition

https://github.com/math-and-data/pandas/blob/44b7de3a5b7541bb2f4fee8a0f4e8f527b4c1b29/pandas/core/generic.py#L5152

Reply:
I'm happy to take a stab at this - always nice when I can remove code too

Reply:
@math-and-data awesome thanks! Can you open a separate issue for this?

Reply:
will do.

Reply:
@WillAyd I was waiting for this PR to be approved, then I would open a new request where I change the relevant code (remove the 'axis' reference) and edit the documentation accordingly. Is there anything else I had missed in this PR (other than the suggestion of breaking out the DataFrame and Series examples)?